.github/workflows/ci-build.yml

```diff
@@ -48,7 +48,6 @@ jobs:
       fail-fast: true
       matrix:
         ember-try-scenario:
-          - ember-lts-3.4
```

Comment:
This seems unrelated, does it cause failures with this API? Ideally we do not need to change our support constraints in order to migrate to the new public API...

Reply:
Removed support for 3.4 in this PR https://github.com/ember-lifeline/ember-lifeline/pull/855

Reply:
Will need to be rebased, since this already landed